src/FlaUI.WebDriver/Keys.cs

```diff
@@ -0,0 +1,505 @@
+﻿using FlaUI.Core.WindowsAPI;
+
+namespace FlaUI.WebDriver;
+
+internal class Keys
+{
+    /// <summary>
+    /// Normalized key mapping from https://www.w3.org/TR/webdriver2/#keyboard-actions
+    /// </summary>
+    private static readonly Dictionary<char, string> s_normalizedKeys = new Dictionary<char, string>()
```

Comment:
The spec maps these unicode key points to a string value, but this would probably be better expressed in C# using an enum.